src/Assets/dotnet-format/Directory.Build.props

```diff
@@ -0,0 +1,10 @@
+<Project>
+
+  <!-- This is an empty Directory.Build.props file to prevent projects which reside
+       under this directory to use any of the repository local settings. -->
```

Comment:
```suggestion
  <!-- This is an Directory.Build.props file to prevent projects which reside
       under this directory to use any of the repository local settings. -->
```